### PR TITLE
Added demo for intrinsic size 

### PR DESCRIPTION
**Pull request by loonybear**
@ebidel PTAL thanks

### CODE REVIEW

File: intrinsic-size/sized.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+</div>
+<div>
+<button id="sizedButton">Get Image Sizing Info</button>
+</div>
+<div id="sizedOutput" class="output">
+  <pre id="log"></pre>
+</div>
+<script>
+ document.querySelector('#sizedButton').addEventListener('click', function() {
+  var img = document.querySelector('#sizedImage');
```
Comment (ebidel) [inline intrinsic-size/sized.html]: use `const` for these

File: intrinsic-size/sized.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  var output = "width = " + img.width + ", height = " + img.height +
```
Comment (ebidel) [inline intrinsic-size/sized.html]: use an template string instead of the concatenation

File: intrinsic-size/sized.html (hunk 1 above)
Comment (ebidel) [reply intrinsic-size/sized.html]: same with the other demos

File: intrinsic-size/sized.html (hunk 2 above)
Comment (ebidel) [reply intrinsic-size/sized.html]: same with the other demos

File: intrinsic-size/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  var demo = button.nextElementSibling;
+  if (demo.style.display === "block") {
+    demo.style.display = "none";
+  } else {
+    demo.style.display = "block";
+  }
+}
+</script>
+
+<style>
```
Comment (ebidel) [inline intrinsic-size/index.html]: can we move the styles to the top?

File: intrinsic-size/index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+</center>
+                                                                     
+<script>
+if ( !(HTMLImageElement.prototype.hasOwnProperty('intrinsicSize')) ) {
+  ChromeSamples.setStatus('Intrinsic Size Attribute is not available.\n' +
+    'Please make sure the "Experimental Web Platform features" flag is enabled.');
+}
+
+function toggleDemo(button) {
+  button.classList.toggle("active");
```
Comment (ebidel) [inline intrinsic-size/index.html]: use single quotes where possible.

File: intrinsic-size/index.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  var demo = button.nextElementSibling;
```
Comment (ebidel) [inline intrinsic-size/index.html]: `const`

**Review by ebidel — CHANGES_REQUESTED** [review]

**Comment by loonybear** [comment]
Done. PTAL.

File: intrinsic-size/index.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+<h3>Background</h3>
+<p>
+&quot;intrinsicSize&quot; attribute tells the browser to ignore the actual intrinsic size
+of the image and pretend it&apos;s the size specified in the attribute. This is useful in
+combination with <a hred="https://github.com/WICG/feature-policy/blob/master/policies/unsized-media.md">&quot;unsized-media&quot; policy</a>
+because it allows for images proportional to window width without causing layout instability.
+Please read <a href="https://github.com/ojanvafai/intrinsicsize-attribute">the explainer for &quot;intrinsicSize&quot; attribute</a> for more details.
+</p>
+<p>
+This attribute overrides the actual intrinsic size of an media element. Specifically, the image would raster at these
```
Comment (ebidel) [inline intrinsic-size/index.html]: of an media -> of a media

File: intrinsic-size/index.html [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+---
+
+<style>
+.demoContent {
+    display: none;
+    overflow: hidden;
+    background-color: #f1f1f1;
+}
+.demoButton {
+    background-color: #777;
```
Comment (ebidel) [inline intrinsic-size/index.html]: 2-space indent on the blocks.

File: intrinsic-size/index.html [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+<p>
+This attribute overrides the actual intrinsic size of an media element. Specifically, the image would raster at these
+dimensions and <a href="https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-naturalwidth">naturalWidth/naturalHeight</a> 
+on images would return the value specified in this attribute. On video elements, the video would raster at this size and
+<a href="https://html.spec.whatwg.org/multipage/media.html#dom-video-videowidth">videoWidth/videoHeight</a> on the video would return the intrinsicsize values.
+</p>
+
+<h3>Case 1</h3>
+<p>if no width/height are otherwise set on an image, then the size is specified by &quot;intrinsicsize&quot;.</p>
+<button class="demoButton" onclick="toggleDemo(this)">See Demo</button>
```
Comment (ebidel) [inline intrinsic-size/index.html]: I was a bit confused on why the demos are hidden by default and remove the toggle? Can we just show them all? There's only 3 :)

File: intrinsic-size/index.html [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+<center class="demoContent">
```
Comment (ebidel) [inline intrinsic-size/index.html]: Oof. Let's not use `<center>`. That's old school! Use CSS `text-align-center` to center things.

File: intrinsic-size/index.html [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+   <img intrinsicsize="250 x 200" src="cat.jpg">
+  {% endhighlight %}
+  {% include_relative unsized.html %}
+</center>
+
+<h3>Case 2</h3>
+<p>If the width is set on an image, then the height is set to maintain the aspect ratio defined in &quot;intrinsicsize&quot;.</p>
+<button class="demoButton" onclick="toggleDemo(this)">See Demo</button>
+<center class="demoContent">
+  <div align="left">
```
Comment (ebidel) [inline intrinsic-size/index.html]: likewise `align="left"` is pretty. Use CSS `text-align:left` if you need to adjust.

File: intrinsic-size/index.html [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+    border: none;
+    text-align: left;
+    outline: none;
+    font-size: 15px;
+ }
+.active, .demoButton:hover {
+    background-color: #555;
+}
+</style>
+
```
Comment (ebidel) [inline intrinsic-size/index.html]: Need this before your html content:

`{% include output_helper.html %}`

File: intrinsic-size/index.html [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+</p>
+<p>
+This attribute overrides the actual intrinsic size of an media element. Specifically, the image would raster at these
+dimensions and <a href="https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-naturalwidth">naturalWidth/naturalHeight</a> 
+on images would return the value specified in this attribute. On video elements, the video would raster at this size and
+<a href="https://html.spec.whatwg.org/multipage/media.html#dom-video-videowidth">videoWidth/videoHeight</a> on the video would return the intrinsicsize values.
+</p>
+
+<h3>Case 1</h3>
+<p>if no width/height are otherwise set on an image, then the size is specified by &quot;intrinsicsize&quot;.</p>
```
Comment (ebidel) [inline intrinsic-size/index.html]: if -> If

File: intrinsic-size/index.html [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+.active, .demoButton:hover {
+    background-color: #555;
+}
+</style>
+
+<h3>Background</h3>
+<p>
+&quot;intrinsicSize&quot; attribute tells the browser to ignore the actual intrinsic size
+of the image and pretend it&apos;s the size specified in the attribute. This is useful in
+combination with <a hred="https://github.com/WICG/feature-policy/blob/master/policies/unsized-media.md">&quot;unsized-media&quot; policy</a>
```
Comment (ebidel) [inline intrinsic-size/index.html]: hred -> href

Comment (ebidel) [reply intrinsic-size/index.html]: policy -> feature policy

**Review by ebidel — CHANGES_REQUESTED** [review]

**Comment by ebidel** [comment]
One question I have about the feature is how it plays with https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit. `object-fit` is intended to maintain aspect ratio.

Maybe you could add 1-2 sentence in the demo description that address that for readers?

**Review by loonybear — COMMENTED** [review]
Done. Please take another look. Thanks!

**Review by ebidel — APPROVED** [review]

**Comment by loonybear** [comment]
Updated description. Ready to merge.

**Pull request merged**